service/fsStorage.go

```diff
@@ -0,0 +1,92 @@
+package service
```

Comment:
Why have you put `fsStorage` implementation into `service` package? It is definitely intended to be in `storage` package. The same with `aesEncoder`. Please refactor it.